mentat/config.py

```diff
@@ -103,21 +103,17 @@ class Config:
         factory=list,
         metadata={"description": "List of glob patterns to exclude from context"},
     )
-    auto_context: bool = attr.field(
```

Comment:
Could we somehow continue to support running `mentat -a` without having to write a token amount?

Reply:
I should've finished #358 by now, but I'd planned to support any of:
`mentat -a`              default auto-context (8k)
`mentat -a 9000`         9k of auto-context
`mentat -a 32000 8000`   32k to LLMFeatureFilter, 8k max auto_features

Thoughts?

Reply:
> mentat -a default auto-context (8k)
mentat -a 9000 9k of auto-context
mentat -a 32000 8000 32k to LLMFeatureFilter, 8k max auto_features

this is how I think it should be called as well

Reply:
Just pushed a commit that sets the default to 2000 if no value is given (but still disabled if -a isn't given, and still takes a user value if a number is given after). I changed the default to 2000 because most of the time whenever I use auto context the vast majority of the tokens are useless, and now with persistent context I think it makes more sense than ever to have the default be lower, but we can move it back up if you want to.

Reply:
> I should've finished #358 by now, but I'd planned to support any of: `mentat -a` default auto-context (8k) `mentat -a 9000` 9k of auto-context `mentat -a 32000 8000` 32k to LLMFeatureFilter, 8k max auto_features
> 
> Thoughts?

Why don't we have a different flag for llm feature filter max compared to auto features? It doesn't make much sense to me to include them both in a single flag, and especially not to swap the numbers when you add a second one.

Reply:
It's concise, does everything we need, and kinda mirrors the actual process - first 32k, then 8k.

Think of it like Python's `range` function; by default the first kwarg (start) is 0, and if called with a single arg it binds to the second kwarg (end).

Reply:
should we add support for 'k'?

Reply:
> It's concise, does everything we need, and kinda mirrors the actual process - first 32k, then 8k.
> 
> Think of it like Python's `range(start, stop, step)` function; by default the first kwarg (start) is 0, and if called with a single arg it binds to the second kwarg (stop).

I was thinking of this function exactly, and I really dislike it haha. Multiple times I've written a for loop with just the end and then realized I needed a step later on, and didn't realize that because I only added one number it still didn't add the step. It is the only function to change the position of it's arguments based on how many you have. I think it's bad design, and I also think it would be bad design for us to change the order of arguments based on how many we have. Sorry for the rant; I really dislike how the range function works.

Reply:
How about
`mentat -a`
`mentat -a 8000`
`mentat -a 8000 `-lff 32000` or `--llm-feature-filter`

Reply:
That sounds good to me (although I would prefer abbreviation to just be -l); this doesn't relate to my PR at all though, right? Should I add this in this PR or are we just discussing this for future reference.

Reply:
Paul I see your point about the range function: you just want to add a step but now you have to add two more args. I don't think that applies here though as we only have 2 arguments, not 3.

I'm ok with an extra `-lff` or `-l` though, this is just an experimental feature for now.

Reply:
> should we add support for 'k'?

I like this but not in this PR

Reply:
Ya, originally I just wanted to keep a default value of `-a` and we've got that so I'm good for this PR.